curation_concerns-models/app/models/concerns/curation_concerns/model_methods.rb

```diff
@@ -0,0 +1,20 @@
+module CurationConcerns
+  module ModelMethods
```

Comment:
Remove this.  I moved/removed these methods already.  Now they come from the lines where GenericFileBehavior calls:

```ruby
    include Hydra::WithDepositor
    include CurationConcerns::Serializers
```